src/main/java/seedu/address/commons/core/AliasMapping.java

```diff
@@ -97,19 +106,32 @@ public boolean isReservedKeyword(String aliasName) {
         case FindIssueCommand.COMMAND_WORD:
         case EditIssueCommand.COMMAND_WORD:
         case DeleteIssueCommand.COMMAND_WORD:
+
+        //====== Alias Commands ======
+        case AliasCommand.COMMAND_WORD:
+        case UnaliasCommand.COMMAND_WORD:
+        case ListAliasCommand.COMMAND_WORD:
             return true;
         default:
             return false;
         }
     }
 
     /**
-     * Check if the command used is an existing alias.
+     * Checks if the command used is an existing alias.
      */
     public boolean isRecursiveKeyword(String commandWord) {
         return mapping.containsKey(commandWord);
     }
 
+    /**
+     * Returns the current number of user-defined aliases.
+     * @return Size of the current mapping.
```

Comment:
Should there be a blank line between description and other tags?

Reply:
Done!